nomic/aws/sagemaker.py

```diff
@@ -157,5 +159,10 @@ def embed_texts(
         response = client.invoke_endpoint(
             EndpointName=sagemaker_endpoint, Body=batch, ContentType="application/json"
         )
-        embeddings.append(parse_sagemaker_response(response))
-    return np.vstack(embeddings)
+        embeddings.extend(parse_sagemaker_response(response))
+
+    return {
+        "embeddings": embeddings,
+        "model": "nomic-embed-text-v1.5",
```

Comment:
If we are using v1.5, we can and should support both binary and matryoshka embeddings

Reply:
ok i can do this in nomic client but I think it would be better to make the change in sagemaker service.
i think merge this in as-is and i'll include those options in separate PR once i figure out how to push updates to marketplace offering.

Reply:
https://linear.app/nomic/issue/NOM-1368/add-matryoshka-and-binary-option-to-sagemaker-service